cconv/tomlCconv.ml

```diff
@@ -0,0 +1,47 @@
+type t = [
+  | `Int of int
+  | `Float of float
+  | `String of string
+  | `Bool of bool
+  | `List of t list
+  | `Assoc of (string * t) list]
```

Comment:
I just don't know how. I had a little conversation about it with author of cconv https://github.com/c-cube/cconv/issues/11